### ISSUE TITLE
[GNUPG:] KEYEXPIRED

### ISSUE THREAD

**Issue by darkwing1** [keepassxreboot/keepassxc#5172]
I just downloaded v2.6.0 and verified with:
`gpg --status-fd 1 --verify KeePassXC-2.6.0-x86_64.AppImage.sig`

I do get VALIDSIG & GOODSIG but im not sure why it **also** shows `[GNUPG:] KEYEXPIRED`
```
[GNUPG:] NEWSIG
[GNUPG:] KEYEXPIRED 1546540992
[GNUPG:] KEY_CONSIDERED BF5A669F2272CF4324C1FDA8CFB4C2166397D0D2 0
[GNUPG:] SIG_ID l/P5Tsx9zD57aw51NJ6CtQUxOLs 2020-07-07 1594147996
[GNUPG:] KEYEXPIRED 1546540992
[GNUPG:] KEY_CONSIDERED BF5A669F2272CF4324C1FDA8CFB4C2166397D0D2 0
[GNUPG:] GOODSIG B7A66F03B59076A8 KeePassXC Release <release@keepassxc.org>
[GNUPG:] VALIDSIG C1E4CBA3AD78D3AFD894F9E0B7A66F03B59076A8 2020-07-07 1594147996 0 4 0 1 10 00 BF5A669F2272CF4324C1FDA8CFB4C2166397D0D2
[GNUPG:] KEYEXPIRED 1546540992
[GNUPG:] KEY_CONSIDERED BF5A669F2272CF4324C1FDA8CFB4C2166397D0D2 0
[GNUPG:] TRUST_UNDEFINED 0 pgp
[GNUPG:] VERIFICATION_COMPLIANCE_MODE 23
```
Please explain why im getting key expired if the key isnt expired? are one of the sub keys expired?

**Comment by darkwing1** [comment]
I tried removing the public key from my keyring and re-imported the latest one https://keepassxc.org/keepassxc_master_signing_key.asc and the KEYEXPIRED error seems to have gone away. My question is, was one of the subkeys expired?  and do i have to import the public key regularly?
I thought `gpg --keyserver hkp://keyserver.ubuntu.com --refresh-keys` would suffice, but it seems that didnt work, so i had to re-import it. (I saw the public key i already had was smaller than the one i imported now) why is this?
